src/App.tsx

```diff
@@ -1,156 +1,91 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
-
-export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+import {
+  FC,
+  useState,
+  useEffect,
+  useCallback,
+  useMemo,
+} from 'react';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Footer } from './components/Footer';
+import { Status } from './enum/Status';
+
+const USER_ID = 10303;
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todoStatus, setTodoStatus] = useState<string>(Status.All);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const showError = (message: string) => {
+    setErrorMessage(message);
+    window.setTimeout(() => {
+      setErrorMessage('');
+    }, 3000);
+  };
+
+  useEffect(() => {
+    try {
+      getTodos(USER_ID).then((data) => {
+        setTodos(data);
+      });
+    } catch {
+      showError('Error');
+    }
+  }, []);
+
+  const filterTodos = useCallback(() => {
+    return todos.filter(({ completed }) => {
+      switch (todoStatus) {
+        case Status.Active:
+          return !completed;
+
+        case Status.Completed:
+          return completed;
+
+        case Status.All:
+        default:
+          return todos;
+      }
+    });
+  }, [todos, todoStatus]);
+
+  const visibleTodos = useMemo(filterTodos, [todos, todoStatus]);
+
+  const handleStatus = useCallback((status: string): void => {
```

Comment:
TS already know about it
```suggestion
  const handleStatus = useCallback((status: string) => {
```